spec/models/converters/base_converter_spec.rb

```diff
@@ -14,15 +14,16 @@
     end
   end
 
+  it 'does not convert strings containing more than forbidden words to nil' do
+    str = 'a string containing null and a period.'
+    expect(subject.convert(str)).to eq(str)
+  end
+
   it 'converts nil to nil' do
     expect(subject.convert(nil)).to be_nil
   end
 
   it 'converts blanks to an empty string' do
     expect(subject.convert('      ')).to be_blank
   end
-
```

Comment:
Instead of removing this test case, why not flip its logic to be "it preserves input case" and expect input to match output. That way if behavior changes later it is caught by the spec.